bundles/org.openhab.binding.edf/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,116 @@
+				<description>Yearly fix price including taxes.</description>
+			</channel>
+			<channel id="variable-ht" typeId="energy-price-ht">
+				<label>Variable Price HT</label>
+				<description>Energy price in €/kWh excluding taxes.</description>
+			</channel>
+			<channel id="variable-ttc" typeId="energy-price-ttc">
+				<label>Variable Price</label>
+				<description>Energy price in €/kWh including taxes.</description>
+			</channel>
+		</channels>
+
+		<config-description>
+			<parameter name="puissance" type="integer" min="3" max="36">
+				<default>6</default>
+				<label>Power output</label>
+				<description>PDL power output (in kVA)</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="hphc">
+		<label>HP-HC Tariff</label>
+		<description>High / Low usage Tariff Option</description>
+
+		<channels>
+			<channel id="tariff-start" typeId="timestamp"/>
+			<channel id="fixed-ht" typeId="price-ht">
+				<label>Fix Price HT</label>
+				<description>Yearly fix price excluding taxes.</description>
+			</channel>
+			<channel id="fixed-ttc" typeId="price-ttc">
+				<label>Fix Price</label>
+				<description>Yearly fix price including taxes.</description>
+			</channel>
+			<channel id="hc-ht" typeId="energy-price-ht">
+				<label>Low Hours HT</label>
+				<description>Low hours energy price in €/kWh excluding taxes.</description>
+			</channel>
+			<channel id="hc-ttc" typeId="energy-price-ttc">
+				<label>Low Hours</label>
+				<description>Low hours energy price in €/kWh including taxes.</description>
+			</channel>
+			<channel id="hp-ht" typeId="energy-price-ht">
+				<label>High Hours HT</label>
+				<description>High hours energy price in €/kWh excluding taxes.</description>
+			</channel>
+			<channel id="hp-ttc" typeId="energy-price-ttc">
+				<label>High Hours Price</label>
+				<description>High hours energy price in €/kWh including taxes.</description>
+			</channel>
+		</channels>
+
+		<config-description>
+			<parameter name="puissance" type="integer" min="3" max="36">
+				<default>6</default>
+				<label>Power output</label>
+				<description>PDL power output (in kVA)</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="price-ht" advanced="true">
+		<item-type>Number:Currency</item-type>
+		<label>Price</label>
+		<category>Price</category>
+		<state readOnly="true" pattern="%.2f %unit%"></state>
+	</channel-type>
+
+	<channel-type id="price-ttc">
+		<item-type>Number:Currency</item-type>
+		<label>Price</label>
+		<category>Price</category>
+		<state readOnly="true" pattern="%.2f %unit%"></state>
+	</channel-type>
+
+	<channel-type id="timestamp" advanced="true">
+		<item-type>DateTime</item-type>
+		<label>Tariff Start</label>
+		<description>Beginning date for this tariff</description>
+		<category>Time</category>
+		<state readOnly="true" pattern="%1$tY-%1$tm-%1$td"/>
+	</channel-type>
+
+	<channel-type id="energy-price-ht" advanced="true">
```

Comment:
What exactly is the difference between price-ht and energy-price-ht ?

Reply:
I got it, the first is for the "abonnement" and is a fixed price, the second is for the consumption.

Reply:
C'est le prix annuel de l'abonnement. energy-price-ht est le prix par kWH

Reply:
J'ai mis du temps à comprendre :)

Maybe the names are not ideal, I am not sure.

Reply:
Names are based on dataset field names - and for sure, seeing them as is in english is a bit weird. We'll be able to adapt this in the french translation once merged for more clarity.